.github/workflows/scorecardtests.yml

```diff
@@ -0,0 +1,33 @@
+name: Scorecard tests
+
+on: [push, pull_request]
+
+jobs:
+
+  sct:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+        
+    - name: Set up Go
```

Comment:
Some of these steps are common with the e2e tests.  Can we setup a new action that we use in this new test and in the e2e tests?  See .github/actions/setup-e2e for an example of the action I created for the e2e tests.